tests/integration/dryrun_mixin_docs_test.py

```diff
@@ -0,0 +1,608 @@
+"""
+Derived from: https://github.com/algorand/docs/blob/bbd379df193399f82686e9f6d5c2bcb9d676d2d7/docs/features/asc1/teal_test.md#basic-setup-and-simple-tests
+"""
```

Comment:
New test, based on old devrel docs. Instroduced mainly to verify that refactoring of `DryRunTestMixin` did not break existing functionality.

Reply:
@algorandskiy - Here's what I was talking about today.